README.md

```diff
@@ -6,20 +6,29 @@ Context-oriented Programming (COP) for JavaScript
 ## Example Code 
 
 ```JS
-import * as cop from "https://lively-kernel.org/lively4/ContextJS/Layers.js"
+import "https://lively-kernel.org/lively4/ContextJS/ContextJS.js"
```

Comment:
It would be awesome if that would work, but unfortunately that is not supported by ES6 - as quick tests have shown my version also doesn't work.. we need to do `import { proceed, withLayers, withoutLayers, layer, Layer } from '../contextjs.js'` - any ideas @JensLincke ?